src/ReaderTaskEither.test.ts

```diff
@@ -14,7 +41,30 @@ import {
   picksTaskEitherK,
   picksTaskK,
   picksW,
+  read,
+  readEither,
+  readIO,
+  readIOEither,
+  readOption,
+  readReader,
+  readReaderEither,
+  readReaderTask,
+  readReaderTaskEither,
+  reads,
+  readsEither,
+  readsIO,
+  readsIOEither,
+  readsOption,
+  readsReader,
+  readsReaderEither,
+  readsReaderTask,
+  readsReaderTaskEither,
+  readsTask,
+  readsTaskEither,
+  readTask,
+  readTaskEither,
 } from './ReaderTaskEither'
+import * as $S from './struct'
 
 describe('ReaderTaskEither', () => {
   describe('pick', () => {
```

Comment:
How would you like to add a test if the given object does not have the _picked_ property?

Reply:
You cannot select a nonexistent property according to the type signature:
```typescript
<R extends $S.Struct>(): <K extends keyof R>(k: K) => ReaderTaskEither<Pick<R, K>, never, R[K]>
```